ansible_base/rbac/management/__init__.py

```diff
@@ -9,19 +9,19 @@
 logger = logging.getLogger(__name__)
 
 
-def create_dab_permissions(app_config, verbosity=2, interactive=True, using=DEFAULT_DB_ALIAS, apps=global_apps, **kwargs):
+def create_dab_permissions(app_config, verbosity=2, interactive=True, using=DEFAULT_DB_ALIAS, apps=global_apps, **kwargs) -> bool:
```

Comment:
For background, this is modeled after

https://github.com/django/django/blob/main/django/contrib/auth/management/__init__.py

To a fault, it was trying to not change any signatures. But in the weeks and months after merging this, it became pretty apparent that we won't be able to keep its scope so limited in the long run. It's still dangerous code to mess with since it'll run with any given model states and has (or will have) dependencies on post-migrate code from multiple other apps (content types, resource registry).